patches/twing+2.3.4.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/node_modules/twing/lib/token-parser/spaceless.js b/node_modules/twing/lib/token-parser/spaceless.js
+index 3c09e1f..269aceb 100644
+--- a/node_modules/twing/lib/token-parser/spaceless.js
++++ b/node_modules/twing/lib/token-parser/spaceless.js
+@@ -16,7 +16,6 @@ const token_1 = require("../token");
+ const spaceless_1 = require("../node/spaceless");
+ class TwingTokenParserSpaceless extends token_parser_1.TwingTokenParser {
+     parse(token) {
+-        console.error('The "spaceless" tag is deprecated since Twig 2.7, use the "spaceless" filter instead.');
```

Comment:
Is there a better way to address the issue instead ?

What prevent you to use the `spaceless` filter instead of the tag ?

(_just for my personnal info_)

Reply:
Our templates are twig-1 implementations, basically :) 
this is mainly due to the drupal version of twig in use, so this check at the moment is not pertinent.

Reply:
Ok fair enough.